e2-studio-star-rx72n-firmware/src/tasks/telemetry_task.c

```
@@ -1852,9 +1847,25 @@ static rx_err_t internal_build_and_send_telemetry(void)
 
+  /* Select transport: USB preferred, SPI fallback */
+  telemetry_transport_t transport = internal_select_transport();
+
+  if (transport == k_telemetry_transport_none) {
+    /* No transport available; drop message this cycle */
+    return k_rx_err_invalid_state;
+  }
+
+  /* Map transport selection to comm manager channel */
+  rx_comm_channel_t channel;
+  if (transport == k_telemetry_transport_usb) {
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)